backend/src/akvo/lumen/lib/import.clj

```diff
@@ -71,8 +71,9 @@
                                                       (assoc import-config :environment (env/all conn)))]
           (let [columns (p/columns importer)]
             (postgres/create-dataset-table conn table-name columns)
-            (doseq [record (map postgres/coerce-to-sql (take common/rows-limit (p/records importer)))]
-              (jdbc/insert! conn table-name record))
+            (doseq [record (take common/rows-limit (p/records importer))]
+              (doseq [i (range (count (last (first record))))]
```

Comment:
What's the _meaning_ of this expression? `(range (count (last (first record))))`? :exploding_head:  Can we improve it? Create a function with a meaningful name?